components/front-page/LearnSection.tsx

```diff
@@ -1,32 +1,49 @@
 import { motion } from "framer-motion";
+import Image from "next/image";
 
 interface LearnCardProps {
   tag: string;
   title: string;
   description: string;
   link: string;
-  className?: string;
+  img: string;
+  classes?: string;
 }
 
 const LearnCard = ({
   tag,
   title,
   description,
-  className,
+  classes,
```

Comment:
think this should have default value set to empty string, otherwise the class would show `undefined` for an element if the parent component doesn't set anything.